.python-version

```diff
@@ -1 +0,0 @@
-3.10.14
```

Comment:
I'm not sure it's safe to delete this file. It was needed for something (I don't remember exactly what)